specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2015-03-01-preview/cluster.json

```diff
@@ -1228,6 +1322,48 @@
                 }
             }
         },
+        "UpdateGatewaySettingsParameters": {
+            "properties": {
+                "restAuthCredential.isEnabled": {
+                    "type": "boolean",
+                    "default": true,
+                    "x-ms-client-name": "isCredentialEnabled",
+                    "description": "Indicates whether or not the gateway settings based authorization is enabled."
+                },
+                "restAuthCredential.username": {
+                    "type": "string",
+                    "x-ms-client-name": "userName",

```

Comment:
AutoRest validation fails due to error `DefinitionsPropertiesNamesCamelCase`.  This error is caused by name `restAuthCredential.isEnabled`, `restAuthCredential.username`and `restAuthCredential.password`. We wonder if it is possible to by pass the error check because these names have existed for a long time. In the past, we just didn't design a model for them.